site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -4,6 +4,26 @@
 [field]#sourceTenantId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.14.0#::
 The optional Id of an existing Tenant to make a copy of. If present, the [field]#tenant.id# and [field]#tenant.name# values of the request body will be applied to the new Tenant, all other values will be copied from the source Tenant to the new Tenant.
 
+[field]#tenant.captchaConfiguration.captchaMethod# [type]#[String]# [required]#Required# [since]#Available since 1.30.0#::
+The type of captcha method to use. The possible values are:
++
+* `GoogleRecaptchaV2` - use Google Recaptcha v2
+* `GoogleRecaptchaV3` - use Google Recaptcha v3
+* `HCaptcha` - use HCaptcha
+* `HCaptchaEnterprise` - use HCaptcha Enterprise
+
+[field]#tenant.captchaConfiguration.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.30.0#::
+Whether captcha configuration is enabled.
+
+[field]#tenant.captchaConfiguration.secretKey# [type]#[String]# [required]#Required# [since]#Available since 1.30.0#::
+The secret key for this captcha method.
+
+[field]#tenant.captchaConfiguration.siteKey# [type]#[String]# [required]#Required# [since]#Available since 1.30.0#::
+The site key for this captcha method.
+
+[field]#tenant.captchaConfiguration.threshold# [type]#[Float]# [optional]#Optional# [default]#defaults to `0.5`# [since]#Available since 1.30.0#::
+The threshold which separates passing from failing the Google v3 or HCaptcha Enterprise score. If not using either of those methods then this value is ignored. The value must be between `0.0` and `1.0`.
```

Comment:
```suggestion
The numeric threshold which separates a passing score from a failing one. This value only applies if using either the Google v3 or HCaptcha Enterprise method, otherwise this value is ignored.
+
The value must be between `0.0` and `1.0`. Values outside of that range will result in an error.
```